src/rules/switchFinalBreakRule.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright 2017 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { endsControlFlow, isBlock, isCaseBlock } from "tsutils";
+import * as ts from "typescript";
+
+import * as Lint from "../index";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    /* tslint:disable:object-literal-sort-keys */
+    public static metadata: Lint.IRuleMetadata = {
+        ruleName: "switch-final-break",
+        description: Lint.Utils.dedent`
+            Enforces consistent use of 'break;' in the final clause of a 'switch' statement.
+            A 'default' clause should never have a 'break'.
+            A 'case' clause should always have a 'break' (or other control flow exit) even if it is the last clause.`,
+        optionsDescription: "Not configurable.",
```

Comment:
consider making this rule configurable with options like "always" and "never" (and if you really want it, you can add another option to handle `default` different. although I don't understand why the default clause should be handled differently at all).

For the record, I'd prefer setting this rule to "never" in this repository to avoid unnecessary statements.